.github/workflows/build-and-push.yml

```diff
@@ -18,7 +18,10 @@ jobs:
       contents: read
       deployments: write
       id-token: write
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
`ubuntu-latest` currently points to `ubuntu-22.04`, we need `ubuntu-24.04` so we can install `just` via `apt`.